packages/tools/create-kadena-app/templates/nextjs/src/pages/index.tsx

```diff
@@ -0,0 +1,120 @@
+import Head from 'next/head'
+import React, { useState } from 'react'
+import { SpinnerRoundFilled } from 'spinners-react'
+import { Pact, signWithChainweaver } from '@kadena/client'
+import styles from '../styles/main.module.css'
+
+const NETWORK_ID = 'testnet04'
```

Comment:
Can we make this build configurable
e.g. `pnpm run build --production` will change this to mainnet or at least look into a `kadena.config` file.

We can of course add this in a later feature.

Reply:
Yes, the current contract is only deployed on testnet, but I can atleast make it configurable and move to config files. Will do that in the PR for the VueJS template I'm currently working on.